Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -74,13 +68,20 @@ So you would be running different isolated consumer clusters on top of single `S
     Restricting the csi-users per pool, and per cluster will require creating new csi-users and new secrets for that csi-users.
     So apply these secrets only to new `Consumer cluster` deployment while using the same `Source cluster`.
 
-#### Sample run
-
 ```console
 python3 create-external-cluster-resources.py --cephfs-filesystem-name <filesystem-name> --rbd-data-pool-name <pool_name> --cluster-name <cluster-name> --restricted-auth-permission true --format <bash> --rgw-endpoint <rgw_endpoin> --namespace <rook-ceph-external>
 ```
 
-#### Upgrade Sample Run
+### RGW Multisite
+
+Pass the --rgw-realm-name, --rgw-zonegroup-name and --rgw-zone-name flags to create the admin ops user in a specific zone, zonegroup and realm.
+See the [Multisite doc](https://docs.ceph.com/en/quincy/radosgw/multisite/#configuring-a-master-zone) for creating a zone, zonegroup and realm.
+
+```console
+python3 create-external-cluster-resources.py --upgrade --rbd-data-pool-name replicapool --cluster-name rookstorage --run-as-user client.csi-rbd-node-rookstorage-replicapool
```

Comment:
This comment looks incorrect to me atleast for RGW usecase. IMO it is better to mention give master zone/zonegroup until https://github.com/ceph/ceph/pull/48731 this got fixed

Reply:
So this command needs the master zone/zonegroup flags added?

Reply:
While creating the zones and zgs they needed master flag